client/azure/azure.go

```diff
@@ -0,0 +1,132 @@
+package azure
```

Comment:
I see this client stuff is just a factory to create new clients. Still fits into the service package for me. My two cents. Enough said. Do what you think is right.